### DISCUSSION TITLE
Incomplete error message after submission

### DISCUSSION THREAD

**Discussion by wangln19** [PaddlePaddle/PaddleSpatial#216] [BaiduKDDCup2022] [answered]
A submission error occurred as shown in the following figure:
![image](https://user-images.githubusercontent.com/96399074/175025192-64aa1b32-8ac2-488e-83bf-ec6758ac40cc.png)
I wonder if I could obtain the whole error message.
Our team name is leyezuTeam, and the submission time: one is 2022-06-22 19:54, another is 2022-06-22 23:40, we want to know the detailed information about these 2 ERROR messages.
It is strange for we successfully run the evaluation.py in our own computer, and it takes the platform about 8-15 min to return the ERROR message. I cannot understand what happened.

**Comment by ethanyxx** [comment, marked as answer]
Hi,

Below is the error log related to your submission at  2022-06-22 19:54.
![截屏2022-06-24 17 33 15](https://user-images.githubusercontent.com/62667125/175509648-4ab77e97-d912-47fb-af98-9d4054134ce1.png)

Your submission at 2022-06-22 23:40 is successful.

Regards,

**Reply by wangln19** [reply]
thank you!
